lib/buffer.js

```diff
@@ -22,19 +22,16 @@
 'use strict';
 
 const binding = process.binding('buffer');
+const config = process.binding('config');
 const { compare: compare_, compareOffset } = binding;
 const { isArrayBuffer, isSharedArrayBuffer, isUint8Array } =
     process.binding('util');
 const bindingObj = {};
 const internalUtil = require('internal/util');
+const pendingDeprecation = !!config.pendingDeprecation;
 
-class FastBuffer extends Uint8Array {
-  constructor(arg1, arg2, arg3) {
-    super(arg1, arg2, arg3);
-  }
-}
+const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

Comment:
So… what this would break, afaik, is that `Buffer.prototype.constructor === Buffer` no longer holds true, which is a bit odd but something we might be able to just accept. Everything else, including `instanceof` and `buf.constructor.name`, should continue to work fine.

One alternative way that was brought up, and I think I might prefer that, we should be able to set `FastBuffer[Symbol.species]` to `FastBuffer`, that should keep the `Uint8Array` methods working, too. (Didn’t try that out, though).